epitator/count_annotator.py

```diff
@@ -99,14 +99,17 @@ class CountAnnotator(Annotator):
     def annotate(self, doc):
         if 'spacy.tokens' not in doc.tiers:
             doc.add_tier(SpacyAnnotator())
-        if 'stanford.times' not in doc.tiers:
-            doc.add_tier(JVMNLPAnnotator([
-                'times', 'nes', 'sentences', 'tokens']))
         counts = []
-        for ne_span in doc.tiers['stanford.nes'].spans:
-            if ne_span.type == 'NUMBER' and is_valid_count(ne_span.text):
+        for ne_span in doc.tiers['spacy.nes'].spans:
+            if ne_span.label in ['QUANTITY', 'CARDINAL'] and is_valid_count(ne_span.text):
                 counts.append(MatchSpan(ne_span, 'count'))
-
+            elif ne_span.label == 'DATE' and is_valid_count(ne_span.text):
```

Comment:
What about other label types, TIME, PERCENT, MONEY? J2 will require us to find funding sources. Might be a good test to see how we are going with respect to funds sooner rather than later. If it's a time value less than a day, should we interpret this as happening on that day? Open questions for thought.